### PR TITLE
[Misc] Fix partially tested method in rendering-configuration-default

### PR DESCRIPTION
**Pull request by surli**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java [hunk 1]
```diff
@@ -98,6 +98,15 @@ public void isImageDimensionsIncludedInImageURL() throws Exception
         assertFalse(this.defaultExtendedRenderingConfiguration.isImageDimensionsIncludedInImageURL());
     }
 
+    @Test
+    public void isImageDimensionsIncludedInImageURL2() throws Exception
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: What does the "2" means? Could we be more explicit in the method name to differentiate it from the other test ?

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: Maybe it should go int he existing test.

Comment (surli) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: Actually this is exactly the same test as above but reversing the values: I'm checking that a mock returning true, returns true... 
I'd be glad to have suggestion if you have a more meaningful name in this case

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: Solution: merge it with the previous one. They're testing the same thing.

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: >  I'm checking that a mock returning true, returns true...

I think it's a bit more complex than that, and that there are intermediary steps no?

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: Just checked and indeed it's more complex than that. It's the "default" configuration source that is mocked.

It also checks that configuration source is called with the right parameter.

Comment (surli) [reply xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-configuration/xwiki-platform-rendering-configuration-default/src/test/java/org/xwiki/rendering/internal/configuration/DefaultExtendedRenderingConfigurationTest.java]: OK so I'll merge it with the previous one.

**Pull request merged**